client/gutenberg/editor/controller.js

```diff
@@ -42,9 +42,13 @@ function getPostID( context ) {
 	return parseInt( context.params.post, 10 );
 }
 
-export const jetpackBlocki18n = ( context, next ) => {
-	if ( ! isEnabled( 'gutenberg/block/jetpack-preset' ) ) {
-		return next();
+export const loadTranslations = ( context, next ) => {
+	const domains = [ 'gutenberg' ];
+	if ( isEnabled( 'gutenberg/block/jetpack-preset' ) ) {
+		/*domains.push( {
```

Comment:
If I uncomment these lines for loading also the Jetpack block translations, I'm getting this error: 
```
332:1 Uncaught (in promise) {gutenberg: {…}, jetpack: {…}}
```
<img width="1272" alt="screen shot 2018-12-03 at 16 48 42" src="https://user-images.githubusercontent.com/1233880/49384456-52323580-f71b-11e8-86e1-621b32b66e38.png">


It looks like a bug in `waitForData` but I'm not able to identify it.